src/pages/ShoppingCart.js

```diff
@@ -0,0 +1,128 @@
+import {faTrashCan} from '@fortawesome/free-solid-svg-icons';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {BiMinus, BiPlus} from 'react-icons/bi';
+import styled from 'styled-components';
+
+import useStore from '../hooks/useStore.js';
+
+export default function ShoppingCart() {
+  const {shoppingItems, removeShoppingItem, minusItemQuantity, plusItemQuantity} = useStore(state => ({
+    shoppingItems: state.shoppingItems,
+    removeShoppingItem: state.removeShoppingItem,
+    minusItemQuantity: state.minusItemQuantity,
+    plusItemQuantity: state.plusItemQuantity,
+  }));
+  function subtotal() {
+    return shoppingItems
+      .map(item => item.price * item.quantity)
+      .reduce((previousValue, currentValue) => previousValue + currentValue, 0);
+  }
+  return (
+    <Main>
+      <h1>Shopping Cart</h1>
+      {shoppingItems.length === 0 ? (
+        <div>Your shopping cart is empty!</div>
+      ) : (
+        <>
+          <CartContainer>
+            {shoppingItems.map(item => {
+              return (
+                <ItemContainer key={item.id}>
+                  <IMG src={item.image} />
```

Comment:
You might want to add an `alt` attribute, even if it is just empty.
```suggestion
                  <IMG src={item.image} alt="" />
```

Reply:
Thanks, I changed it.